internal/service/httpserver.go

```diff
@@ -125,8 +133,64 @@ func (b *HttpServer) BootstrapHandler(
 			b.isRunning = false
 		}()
 
+		var ln net.Listener
+		listenMode := strings.ToLower(b.cfg.Service.SecurityOptions[bscfg.SecurityModeKey])
+		switch listenMode {
+		case zerotrust.ZeroTrustMode:
+			ozUrl := b.cfg.Service.SecurityOptions["OpenZitiController"]
+			var zctx ziti.Context
+			var authErr error
+			if os.Getenv("SERVICE_SECURITYOPTIONS_OPENZITIAUTHMETHOD") == "identity" {
```

Comment:
this should eventually  probably be replaced by using the standard configuration options but these two keys need to be added to the "default" and I can't remember where that is :)